setup.py

```diff
@@ -5,15 +5,18 @@
 # Use of this source code is governed by a BSD-3-clause license that can be
 # found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
 
-import imp
+import importlib.util
```

Comment:
I remember this has been fixed in https://github.com/apple/coremltools/pull/2170? (thx for that great PR btw 👍) Please rebase on top of current main

Reply:
updated my fork to reflect those changes